public/stylesheets/style.css

```diff
@@ -0,0 +1,24 @@
+
+* {
+  border-radius: 0 !important;
+}
+
+.splash-background {
```

Comment:
Please remove splash-background - we need to add the process that will take care of browser prefixing instead of doing it manually in the CSS